inji-verify-test/src/test/java/utils/BaseTest.java

```diff
@@ -1,25 +1,19 @@
 package utils;
 
-import org.json.JSONObject;
-
 import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserContext;
 import com.microsoft.playwright.BrowserType;
 import com.microsoft.playwright.Page;
 import com.microsoft.playwright.Playwright;
 
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 
-public class BaseTest {
-
+public class BaseTest{
+	
 	public static Playwright playwright;
 	public static Browser browser;
-
-	protected static final String ENVIRONMENT = System.getProperty("env") == null ? "qa-inji"
-			: System.getProperty("env");
-
-	BaseTestUtil baseTestUtil = new BaseTestUtil();
-
```

Comment:
Don't delete these lines as these are required to read the url's from config file.